charlib/liberty/liberty.py

```
@@ -216,7 +288,8 @@ def to_liberty(self, indent_level=0, precision=1, **kwargs) -> str:
 class Define(Attribute):
     """A define is basically a complex attribute with a bit more input validation"""
     def __init__(self, attribute_name, group_name, attribute_type):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)